config/acme/machines/config_pio.xml

```diff
@@ -28,13 +28,11 @@
     </values>
   </entry>
 
-  <!--- uncomment and fill in relevant sections
   <entry id="PIO_ROOT">
     <values>
-      <value></value>
```

Comment:
This is enough to change each model's ROOT?

Reply:
yes, I confirmed it with a quick test

```
./xmlquery PIO_ROOT
	PIO_ROOT: ['CPL:0', 'ATM:0', 'LND:0', 'ICE:0', 'OCN:0', 'ROF:0', 'GLC:0', 'WAV:0', 'ESP:0']
```